lms/envs/common.py

```diff
@@ -199,21 +193,17 @@
     # .. toggle_description: None
     # .. toggle_use_cases: open_edx
     # .. toggle_creation_date: 2013-04-13
-    # .. toggle_target_removal_date: None
-    # .. toggle_warnings: None
-    # .. toggle_tickets: None
     'ENABLE_MASQUERADE': True,
 
     # .. toggle_name: FEATURES['ENABLE_SYSADMIN_DASHBOARD']
     # .. toggle_implementation: DjangoSetting
     # .. toggle_default: False
     # .. toggle_description: enables dashboard at /syadmin/ for django staff, for seeing overview of system status, for
     #   deleting and loading courses, for seeing log of git imports of courseware. Note that some views are noopen_edx
-    # .. toggle_use_cases: open_edx
+    # .. toggle_use_cases: temporary, open_edx
```

Comment:
**Note:** I believe `toggle_use_cases` is still confusing to people, which will result in loss of the most important data. Noting the `toggles_warnings`, it is clear that this is meant to be a temporary toggle, and knowing when a toggle is supposed to be temporary is probably the primary piece of data from this annotation. Note that `toggle_target_removal_date` is not a great indicator, because many people seem afraid to commit to any date.

This comment is to be used for a future discussion on the best annotation name and values to capture this info.